src/timePicker/TimePicker.tsx

```diff
@@ -2,18 +2,6 @@ import React, { PureComponent } from 'react'
 import cc from 'classcat'
 import prefix from '_utils'
 
-/**
- * Whether or not Date#toLocaleTimeString supports arguments `locales` et `options`.
- */
-const toLocaleTimeStringSupportsLocales = (() => {
```

Comment:
Not used?

Reply:
Seems not, it was a raised warning by the linter.